robustness/tools/breeds_helpers.py

```diff
@@ -10,35 +11,48 @@ class ClassHierarchy():
     '''
     Class representing a general ImageNet-style hierarchy.
     '''
-    def __init__(self, info_dir):
+    def __init__(self, info_dir, root_wnid='n00001740'):
         """
         Args:
             info_dir (str) : Path to hierarchy information files. Contains a 
                 "class_hierarchy.txt" file with one edge per line, a
                 "node_names.txt" mapping nodes to names, and "class_info.json".
         """
         
-        with open(f'{info_dir}/class_info.json', 'r') as f:
+        # Details about dataset class (sleaves) names, IDS
+        with open(f'{info_dir}/dataset_class_info.json', 'r') as f:
```

Comment:
Can we have a user-friendly error (maybe with a link to the docs and the breeds repo) if the modified hierarchy file is not found?